docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+
+### 10. Check Game Servers
+Let's make sure that we have one or more Game Servers in a ready state by running this command:
+```
+kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+```
+
+For a fleet of 2 replicas, you should see 2 Game Servers with a Status of Ready:
+```
+NAME                     STATUS    IP              PORT
+simple-udp-thbvz-mqntp   Ready     35.196.249.83   [map[name:default port:7374]]
+simple-udp-thbvz-smchl   Ready     35.229.96.214   [map[name:default port:7016]]
+```
+
+If there is no fleet, please review [Create Game Server Fleet](../docs/create_fleet.md).
+
+
+### 11. Allocate a Game Server
+The service uses Basic Auth to provide some security as to who can allocate gameserver resources.  A generated key is in main.go that you can change.
+
+Now that the ingress has been created and we know it is healthy, let's allocate a Game Server by passing in our user and key to the /address endpoint.
+
+Allocate a Game Server and retrieve its IP address and port by running this command:
+```
+curl -k -u v1GameClientKey:EAEC945C371B2EC361DE399C2F11E https://[the.ip.address]/address
+```
+
+The output should show an IP address and port, similar to this:
+```
+{"address":"35.229.96.214","port":7016}
+```
+
+Check the Game Servers again, and notice the Allocated Status.  You should see something like this:
+```
+NAME                     STATUS      IP              PORT
+simple-udp-thbvz-mqntp   Ready       35.196.249.83   [map[name:default port:7374]]
+simple-udp-thbvz-smchl   Allocated   35.229.96.214   [map[name:default port:7016]]
+```
+
+
+### 12. Make the IP Address Permanent
+Since the ephemeral IP address previously created will only live for as long as the service lives, allocating a permanent IP address is necessary for longer term reliability.
+
+:warning: This step may cause additional billing charges. :warning:
+
+Reserve a static external IP address named allocator-static-ip by running:
+```
+gcloud compute addresses create allocator-static-ip --global
+```
+
+Configure the existing Ingress to use the reserved IP address by adding an annotation to allocator-ingress.yaml:
+```
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "allocator-static-ip"
+```
+
+Apply the change by running:
+```
+kubectl apply -f allocator-ingress.yaml
+```
+
+
+### 13. Create an A Record
+Setting up a DNS A record that points to the allocator service's permanent IP address is a flexible way to call it from other services or clients.
+
+This step requires that you have a registered domain available.  Use the permanent IP address created previously when creating a new A record with your registrar, and follow your registrars documentation on adding an A record.  When finished, a subdomain, such as allocator-service, should exist.
+
+Now https://allocator-service.yourdomain.ext should return the IP address and port of the allocated Game Server.
+
+
+### 14. Customize
```

Comment:
We could do this as a "next steps" section, i.e.

--- 

Things you could do next
 - Change the fleetname
- Add a SSL certificate
- Change the HTTP password 

I don't think we need to explain how Docker works either. This would also be simpler witha multi-stage build

Reply:
That's a nice way of compressing those last several steps.  I'll make a next steps section while purging the detailed content.